From 5c72f0e97e64a4ab468eee8364005395b255ab3b Mon Sep 17 00:00:00 2001
From: Zhipeng Gong <zhipeng.gong@intel.com>
Date: Fri, 13 Jan 2017 04:09:18 -0500
Subject: [PATCH 148/153] [VPG]: drm/i915: Export global reset_count

Export global reset_count so that the driver can detect the GPU hang
through ioctl.
---
 drivers/gpu/drm/i915/intel_uncore.c |    5 +----
 1 files changed, 1 insertions(+), 4 deletions(-)

diff --git a/drivers/gpu/drm/i915/intel_uncore.c b/drivers/gpu/drm/i915/intel_uncore.c
index 23e9e06..fb3c868 100644
--- a/drivers/gpu/drm/i915/intel_uncore.c
+++ b/drivers/gpu/drm/i915/intel_uncore.c
@@ -1359,10 +1359,7 @@ int i915_get_reset_stats_ioctl(struct drm_device *dev,
 	}
 	hs = &ctx->hang_stats;
 
-	if (capable(CAP_SYS_ADMIN))
-		args->reset_count = i915_reset_count(&dev_priv->gpu_error);
-	else
-		args->reset_count = 0;
+	args->reset_count = i915_reset_count(&dev_priv->gpu_error);
 
 	args->batch_active = hs->batch_active;
 	args->batch_pending = hs->batch_pending;
-- 
1.7.1

